### ISSUE TITLE
CI/CD

### ISSUE THREAD

**Issue by resurrexi** [KalleHallden/BookOmatic#39]
Considering this is an open source project, what are we using for continuous integration/development? We should probably set up the CI/CD pipeline and choose the testing framework, e.g pytest, tox

**Comment by brianraila** [comment]
Github Actions works pretty well here. (CI/CD) Someone else can advice on the testing framework.

**Comment by yasir-khilji-64** [comment]
Github Actions are good. For testing, either pytest or unittest

**Comment by Kieran7741** [comment]
+1 for unittest

**Comment by yasir-khilji-64** [comment]
> +1 for unittest

Yeah it is easy. But I guess, django uses tox and pytest only. *DON'T QUOTE ME ON THIS. Still searching *.

**Comment by robsmithw** [comment]
unittest is the way to go for python test.

**Comment by yasir-khilji-64** [comment]
Well let's start the backend dev for it

**Comment by AlbertCoolGuy** [comment]
How can I assign myself to this issue?

**Comment by lilTrigger619** [comment]
But it seems django already has it's own test module.
What do you think?
